generic/tests/cfg/netperf.cfg

```diff
@@ -118,3 +120,9 @@
                     cygwin_start = C:\rhcygwin\Cygwin.bat -i /Cygwin-Terminal.ico -
                     netserv_start_cmd = netserver
                     netperf_install_cmd = cd netperf-2.6.0; ./configure --enable-burst --enable-demo=yes; make; make install
+    variants:
+        - jumbo:
+            mtu = 9000
+            # please config physical nic name of client for jumbo frame case by uncommenting it
+            # client_physical_nic = <your_client_physical_nic>
```

Comment:
When run the case, why do we need to specify the client nic by ourself? Can't it be getted from code like getting nic of guest and host?

Reply:
can not like we can not know the client ip.

Reply:
You can put it in internal cfg file, like parameter "client".

Reply:
I will add it, anyway it's staf issue not a issue block this.